hadoop-ozone/dist/src/main/compose/ozone-mr/hadoop27/docker-config

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CORE-SITE.xml_fs.AbstractFileSystem.o3fs.impl=org.apache.hadoop.fs.ozone.OzFs
```

Comment:
What's wrong with `OzFS`? Based on my understanding `ozonefs-hadoop2` project includes an `OzFs` implementation which is the same as `BasicOzFs`.

Is there any motivation behind this change?